data/map planets.txt

```diff
@@ -5257,6 +5378,10 @@ planet Valhalla
 	description `	The shipyards of Lionheart Industries are located here. Centuries ago, warships built here turned the tide of the Alpha War, and today the ships designed and built in the Deep continue to be some of the most technologically advanced ships in human space - and the most expensive.`
 	spaceport `Surrounding the spaceport is a metropolis of skyscrapers and factories that spreads all the way to the horizon in every direction.`
 	spaceport `	The spaceport itself is an imposing building, composed of three towers rising over a hundred stories into the air and connected at various levels by bridges. The docking bays are on the higher levels, with warehouse space lower down and loading platforms for trucks at ground level. Massive cargo elevators run up and down in the center of each building, and the hallways are all wide enough to allow small robotic carts and forklifts in addition to foot traffic.`
+	spaceport `	Despite the Deep's importance to defeating the Alphas, Remembrance Day is seldom observed in the region. The only thing you see out of the ordinary is a milk dish on the counter of an information booth.`
```

Comment:
What's a milk dish?

Reply:
It's a tradition of the people of the Deep to put out a dish of milk for the "elves". Like how kids put out cookies and milk for Santa.